README.md

```diff
@@ -42,7 +42,9 @@ Please see [The official documentation](https://www.vaultproject.io/docs/config/
 
 * `purge_config_dir`: Whether the `config_dir` should be purged before installing the generated config.
 
-* `install method`: Supports the values `repo` or `archive`. See [Installation parameters](#installation-parameters).
+* `install_method`: Supports the values `repo` or `archive`. See [Installation parameters](#installation-parameters).
+
+* `manage_unzip`: Whether to manage the `unzip` package on Debian/Ubuntu systems. (defaults to true if `install_method` is set to `archive`)
```

Comment:
I still hold the same position as the `archive` module maintainers. I understand that it "just works" for first time users of the module but it's just more complexity that doesn't really need to be in the module:

 * package name `unzip` isn't necessarily global across all distros
 * if the user tries to install from a tarball archive they will now also need to manually manage that package externally as well as disabling `manage_unzip`.

So including the option actually creates a combination of new configurations users need to be aware of.